src/app/components/fees-row/components/fees-row.layout.tsx

```diff
@@ -0,0 +1,86 @@
+import { Suspense } from 'react';
+
+import { Stack, StackProps } from '@stacks/ui';
+import { useField } from 'formik';
+
+import { CryptoCurrencies } from '@shared/models/currencies.model';
+import { FeeTypes } from '@shared/models/fees/_fees.model';
+import { BitcoinFeeEstimate } from '@shared/models/fees/bitcoin-fees.model';
+import { StacksFeeEstimate } from '@shared/models/fees/stacks-fees.model';
+import { Money } from '@shared/models/money.model';
+
+import { SpaceBetween } from '@app/components/space-between';
+import { SponsoredLabel } from '@app/components/sponsored-label';
+import { Caption } from '@app/components/typography';
+import { WarningLabel } from '@app/components/warning-label';
+
+import { CustomFeeField } from './custom-fee-field';
+import { FeeError } from './fee-error';
+import { FeeEstimateSelect } from './fee-estimate-select';
+import { TransactionFee } from './transaction-fee';
+
+interface FeesRowLayoutProps extends StackProps {
+  feeCurrencySymbol: CryptoCurrencies;
+  feeEstimates: BitcoinFeeEstimate[] | StacksFeeEstimate[];
+  feeInUsd: Money | null;
+  fieldWarning?: string;
+  isCustom: boolean;
+  isSponsored: boolean;
+  isVisible: boolean;
+  onSelectItem(index: number): void;
+  onSetIsSelectVisible(value: boolean): void;
+  onWarning(value: string): void;
+  selected: number;
+}
+export function FeesRowLayout(props: FeesRowLayoutProps) {
```

Comment:
Imo this component isn't just a layout component. It accepts a load of props just to pass down to it's logic-containing children. Per convention, a dumb/layout component shouldn't have logic-containing children or interactive form elements.


Instead, we should pass in the JSX of these elements. This way `FeesRowLayout` is only concerned with where the components go.

```ts
<FeeRowLayout
  selectInput={
    <FeeEstimateSelect
      isVisible={isVisible}
      items={feeEstimates}
      onSelectItem={onSelectItem}
      onSetIsSelectVisible={onSetIsSelectVisible}
      selected={selected}
    />
  }
  feeField={isCustom
    ? <CustomFeeField lowFeeEstimate={feeEstimates[FeeTypes.Low]} setFieldWarning={onWarning} />}
    : <TransactionFee {...} />
/>
```